intro-to-javascript-testing.md

```diff
@@ -0,0 +1,142 @@
+# Intro to JavaScript Testing
+
+* [The right style](#style)
+* [Test your syntax with jshint](#jshint)
+* [Test your logic with jasmine](#jasmine)
+
+## The right style
+
+JavaScript, as a language itself, does not have too many opinions. Functional, reliable, "correct" JS comes in a variety of colors and shapes: in fact, the number of ways to [declare a function](https://www.bryanbraun.com/2014/11/27/every-possible-way-to-define-a-javascript-function/) reaches double digits.  
+
+Its flexibility can cause confusion, headaches, and pure rage. JS invites hard-to-answer questions, like, "Am I doing this right?", "Am I doing this the best way possible?" – and the "right way" may encompass enough solutions to ruin an afternoon. 
```

Comment:
I'd tack the "pure rage" sentence on to the end of the previous paragraph and scrap the rest of this one.